droughtpipeline/load.py

```diff
@@ -0,0 +1,872 @@
+                )
+            body = {
+                "countryCodeISO3": country,
+                "triggersPerLeadTime": triggers_per_lead_time,
+                "disasterType": disasterType,
+                "eventName": event_name,
+                "date": upload_time,
+            }
+            self.ibf_api_post_request("event/triggers-per-leadtime", body=body)
+
+            # END OF EVENT LOOP
+            ###############################################################################################################
+
+        # flood extent raster: admin-area-dynamic-data/raster/floods
+        self.rasters_sent = []
+        for lead_time in set(lead_times_list):
+            drought_extent_new = drought_extent.replace(".tif", f"_{lead_time}_month_{country}.tif" )
+            if lead_time in triggered_lead_times:
+                shutil.copy(
+                    drought_extent.replace(".tif", f"_{lead_time}.tif"), drought_extent_new
+                )
+                
+            else:
+                logging.info(f"no trigger for lead time {lead_time}")
+                #shutil.copy(drought_extent.replace(".tif", f"_empty.tif"),  drought_extent_new, )
+            self.rasters_sent.append(drought_extent_new)
+            files = {"file": open(drought_extent_new, "rb")}
+            self.ibf_api_post_request( "admin-area-dynamic-data/raster/drought", files=files )
+
+
+
+        # send empty exposure data
+        if len(processed_pcodes) == 0:
+            indicators = [
+                "population_affected",
+                "population_affected_percentage",
+                #"alert_threshold",
+                "forecast_trigger",
+                "forecast_severity",
+
+            ]
+            for indicator in indicators:
+                for adm_level in forecast_data.adm_levels:
+                    exposure_pcodes = []
+                    for pcode in forecast_data.get_pcodes(adm_level=adm_level):
+                        if pcode not in processed_pcodes:
+                            amount = None
+                            if indicator == "population_affected":
+                                amount = 0
+                            elif indicator == "population_affected_percentage":
+                                amount = 0.0
+                            #elif indicator == "alert_threshold":
+                            #    amount = 0.0
+                            elif indicator == "forecast_trigger":
+                                amount = 0
+                            elif indicator == "forecast_severity":
+                                amount = 0
+                            exposure_pcodes.append(
+                                {"placeCode": pcode, "amount": amount}
+                            )
+                    body = {
+                        "countryCodeISO3": country,
+                        "leadTime": "1-day",  # this is a specific check IBF uses to establish no-trigger
```

Comment:
This is a floods-remnant. Can you just take the same leadTime that is defined in config.yaml per month?

Reply:
check!